creepsy/README.md

```diff
@@ -0,0 +1,36 @@
+# Creepsy! 
+A scary drink server robot using the Viam platform. This script is derived from the [Tipsy tutorial](https://docs.viam.com/tutorials/projects/tipsy/) but uses an [*audioout* modular resource](https://github.com/viam-labs/audioout) to play scary sound files.
+
+To run this:
+* Set up the audioout module and configure a service named 'audioout':
+
+``` json
+    {
+      "attributes": {},
+      "name": "audioout",
+      "type": "audioout",
+      "namespace": "viam-labs",
+      "model": "viam-labs:audioout:pygame"
+    }
+```
+
+* Upload the effdet0.tflite model and the labels.txt file, and set it up as an mlmodel and vision detector
+* Set environment variables (see note below)
+* run `pip3 install -r requirements.txt` in this directory
+* run `python creepsy.py`
+
+**Required environment variables:**
```

Comment:
What could be cool is if they could point to a directory with scary sound files and it plays one at random from that directory when the triggering event occurs.  Not sure if that needs to be an env variable they point to the directory with